pkg/bundle/sync.go

```diff
@@ -64,6 +63,16 @@ const (
 	fieldManager             = "trust-manager"
 )
 
+type pkcs12Encoder struct {
+	bundlePasswd string
+	bundleData   string
+}
+
+type jksEncoder struct {
+	bundlePasswd []byte
```

Comment:
s.Store() accept second argument  (password) as []byte, so I wanted to avoid type conversion, of course it is not consume much CPU/Memory resources in our case, despite of it is running in container.

Reply:
Ok, makes sense. I would prefer readability of code in this case.